include/lemlib/chassis/differential/differentialSteerRequest.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "pros/motor_group.hpp"
+#include <memory>
+
+namespace lemlib {
+class DifferentialSteerRequest {
```

Comment:
Wtf is this? (add comments)

Reply:
a pointer to this base class is returned by a child class of DifferentialMotion. You pass the motors of the drivetrain to its apply function, and this function would be called by the motion handler. It keeps implementations of the DifferentialMotion class simple when using more complex motor control algorithms, such as a PID controlling motor velocity.

Reply:
ill add documentation